### PR TITLE
feat(illustrations): first release

### PR DESCRIPTION
**Pull request by fabienhebert**
## Type

- Feature

### Summarise concisely:

Create a new "illustration" package with some product illustrations to begin with

### Usage

```ts
import { cockpitOriginal } from "@ultraviolet/illustrations/products/cockpit"
...
<img src={cockpitOriginal} />
```

## Relevant logs and/or screenshots

Capture d’écran 2023-10-09 à 17 30 20

### CODE REVIEW

File: packages/illustrations/.jest/helpers.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const Wrapper = ({ theme = defaultTheme, children }: WrapperProps) => (
+  <ThemeProvider theme={theme}>{children}</ThemeProvider>
+)
+
+export const {
+  renderWithTheme,
+  shouldMatchEmotionSnapshot,
+  shouldMatchEmotionSnapshotWithPortal,
+} = makeHelpers(Wrapper, { classNameReplacer: className => className })
```
Comment (matthprost) [inline packages/illustrations/.jest/helpers.tsx:19]: I don't think we need this helper as the library doesn't need unit testing

**Review by matthprost — CHANGES_REQUESTED** [review]
In the file `codecov.yml` can you add `packages/illustrations`? so we don't have the code coverage counted on those files

File: packages/illustrations/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import products from './products'
+
+export default {
+  products,
```
Comment (matthprost) [inline packages/illustrations/src/index.ts:4]: I don't know if you saw but the ticket got new comment in the structure, we need to have `product.instance`, `product.database` etc

**Review by matthprost — CHANGES_REQUESTED** [review]

File: codecov.yml [hunk 3]
```diff
@@ -20,11 +20,12 @@ parsers:
...
   behavior: default
   require_changes: no
 
 ignore:
-  - "packages/themes"
-  - "packages/ui/src/theme/index.ts"
-  - "examples"
+  - 'packages/themes'
+  - 'packages/ui/src/theme/index.ts'
+  - 'package/illustrations'
```
Comment (matthprost) [inline codecov.yml:30]:
```suggestion
  - 'packages/illustrations'
```

**Review by matthprost — CHANGES_REQUESTED** [review]

**Review by matthprost — APPROVED** [review]

**Review by JulienSaguez — APPROVED** [review]

**Pull request merged**